concourse/pipelines/template/depls-pipeline.yml.erb

```diff
@@ -274,9 +274,9 @@ jobs:
     input_mapping: { scripts-resource: cf-ops-automation, secrets: secrets-<%=depls %>-limited }
     file:  cf-ops-automation/concourse/tasks/bosh_cancel_all_tasks/task.yml
     params:
-      BOSH_CLIENT: ((bosh-client))
-      BOSH_CLIENT_SECRET: ((bosh-client-secret))
-      BOSH_ENVIRONMENT: ((bosh-environment))
+      BOSH_CLIENT: ((bosh-username))
+      BOSH_CLIENT_SECRET: ((bosh-password))
+      BOSH_ENVIRONMENT: ((bosh-target))
```

Comment:
I solved this by overwriting the BOSH_ENVIRONMENT when I run the bosh commands.

https://github.com/orange-cloudfoundry/cf-ops-automation/pull/193/files#diff-283088bd06d72ece4a57d294143738feR42
https://github.com/orange-cloudfoundry/cf-ops-automation/pull/193/files#diff-283088bd06d72ece4a57d294143738feR56
https://github.com/orange-cloudfoundry/cf-ops-automation/pull/193/files#diff-283088bd06d72ece4a57d294143738feR88

Reply:
Good, you've inlined the DNS to IP conversion. But I'm not sure it's a good idea to override the value for maintenance reason, as it will be the only bosh based task to work like this. I'd prefer using `BOSH_TARGET` as external parameter to outline an operation is done on `BOSH_DEPLOYMENT`. rather than an hidden change. What do you think ?

Reply:
I wanted to stray away from BOSH CLI v2 conventions but you might be right.